graylog2-web-interface/src/components/lookup-tables/adapters/index.js

```diff
@@ -1,8 +1,12 @@
 import { PluginManifest, PluginStore } from 'graylog-web-plugin/plugin';
 
+import {} from 'components/maps/adapter';
```

Comment:
This is why the build fails on CI. It looks like this is an unrelated change that snuck in by accident.

Reply:
Good eye!! Thanks. graylog.conf was also erroneously committed. I've removed both. Let's see if the CI builds successfully now.